pom.xml

```
@@ -24,15 +24,7 @@
 		<changelist>-SNAPSHOT</changelist>
 		<gitHubRepo>jenkinsci/test-results-analyzer-plugin</gitHubRepo>
 		<!-- Baseline Jenkins version you use to build the plugin. Users must have this version or newer to run. -->
-		<jenkins.version>2.346.3</jenkins.version>
-		<!-- Jenkins Test Harness version you use to test the plugin. -->
-		<!-- For Jenkins version >= 1.580.1 use JTH 2.x or higher. -->
-		<!-- <jenkins-test-harness.version>2.13</jenkins-test-harness.version> -->
-		<!-- Do not need to mention specifically as maintained by parent dependencies, here used : 2.71 -->
-		<!-- Other properties you may want to use:
-             ~ hpi-plugin.version: The HPI Maven Plugin version used by the plugin..
-             ~ stapler-plugin.version: The Stapler Maven plugin version required by the plugin.
-        -->
+		<jenkins.version>2.361.1</jenkins.version>
```

The recommendation from the ["Choosing a Jenkins version" page](https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/) for developers is for the last release in the LTS line rather than the first release in the LTS line
```suggestion
		<jenkins.version>2.361.4</jenkins.version>
```